mlibrary/search
# [LIBSEARCH-957] Shelf Carousel implementation

# Overview
When viewing a full catalog record, if a call number exists, a carousel appears above the MARC table that shows the next or previous 22 records in the same order as you would see in Catalog Browse.

This pull request closes [LIBSEARCH-957](https://mlit.atlassian.net/browse/LIBSEARCH-957).

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Check out a full catalog record that has a call number.
  - Does a carousel appear?
  - Can you scroll through it?
  - Does the amount of items increase/decrease per slide depending on window size?
  - Can you always return to the current item?
